### PR TITLE
feature json support likes

### PR DESCRIPTION
**Pull request by malinfeng**
- [ ] Do only one thing
- [ ] Non breaking API changes
- [ ] Tested

### CODE REVIEW

File: json.go [hunk 1]
```diff
@@ -162,13 +171,19 @@ func (jsonQuery *JSONQueryExpression) Build(builder clause.Builder) {
...
 				if len(jsonQuery.keys) > 0 {
 					builder.WriteString("JSON_EXTRACT(")
 					builder.WriteQuoted(jsonQuery.column)
 					builder.WriteByte(',')
 					builder.AddVar(stmt, jsonQueryJoin(jsonQuery.keys))
-					builder.WriteString(") = ")
+					if jsonQuery.equals {
+						builder.WriteString(") = ")
+					} else {
+						if jsonQuery.likes {
```
Comment (jinzhu) [inline json.go:183]: Change to `else if jsonQuery.likes`

Comment (malinfeng) [reply json.go]: done

**Comment by malinfeng** [comment]
unit test image cannot start

**Comment by jinzhu** [comment]
It looks like the case branch hasn't been removed.

**Pull request closed without merging**